indico/modules/rb_new/client/js/common/timeline/DailyTimelineContent.jsx

```diff
@@ -145,38 +145,39 @@ export default class DailyTimelineContent extends React.Component {
         const labelWidth = longLabel ? 200 : 150;
 
         return (
-            <div styleName="timeline-content"
-                 className={!selectable && 'timeline-non-selectable'}
-                 style={{width}}>
-                <div style={{
-                    left: labelWidth,
-                    width: `calc(100% - ${labelWidth}px)`,
-                }}
-                     styleName="timeline-lines">
-                    {this.renderDividers(hourSpan, hourStep)}
+            <>
+                <div styleName="timeline-content"
+                     className={!selectable && 'timeline-non-selectable'}
```

Comment:
Won't this end up in `className=false`? IIRC React doesn't like those.

Reply:
Hm, I can change it. I didn't look at it since I didn't modify the code. It has just been moved.